letters/original/15-letters.adoc

```diff
@@ -372,14 +387,19 @@ Calmly therefore, warm at heart, with brotherly love,
 with hopes full of immortality for thee, and for myself,
 I dearly salute thee and thy spouse, remaining Thy affectionate friend,
```

Comment:
same thing as modernized. 1) put comma after "remaining", bump down "Thy affectionate friend," and mark it a SSCL